modules/xfem/doc/content/source/materials/ADLevelSetBiMaterialRankFour.md

```diff
@@ -0,0 +1,27 @@
+# ADLevelSetBiMaterialRankFour
+
+!syntax description /Materials/ADLevelSetBiMaterialRankFour
+
+## Description
+
+This material, `ADLevelSetBiMaterialRankFour` determines the global material
+property by switching the two AD Rank-4 tensor material properties with
+different base_name based on the level set values. While the non-AD equivalent,
+[LevelSetBiMaterialRankFour](LevelSetBiMaterialRankFour.md), is used to compute
```

Comment:
I'd move this sentence to a separate paragraph after this one. It's kind of a different topic, and is a bit confusing in the middle of this paragraph.